File: day05/implode.py
def implode(string):
    chars = ''
    for s in string:
        if s not in chars:
            chars += s
    
    return chars

def get_string():
    string = input('Please enter a string: ')
    
    try:
        print(implode(string))
    except Exception as error:
        print('Error:', error)

if __name__ == '__main__':
    get_string()

File: day05/implode.js
function implode(list) {
  var alpha = 'abcdefghijklmnopqrstuvwxyz';
  var imploded = [];
  
  for (var l in list) {
    var chars = '';
    for (var a in alpha) {
      if (list[l].includes(alpha[a]) && !chars.includes(alpha[a])) {
        chars += alpha[a];
      }
    }
    imploded.push(chars);
  }
  
  return imploded;
}

function getStringToImplode() {
  var strings = prompt('Please enter a list of strings separated by a comma:');
  
  try {
    var stringList = strings.toLowerCase().replace(/\s/g, '').split(',');
    alert(implode(stringList));
  } catch(error) {
    console.log(error);
  }
}
